sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/pipeline/data/expression_component_template.yml

```diff
@@ -10,7 +10,7 @@ outputs:
 
 code: ./src
 
-environment: azureml:AzureML-MLDesigner@latest
+environment: azureml://registries/azureml/environments/mldesigner/labels/latest
```

Comment:
Do we have a job link which we can share?

Reply:
https://ml.azure.com/experiments/id/4daf08b1-e179-4a45-ad11-89e362d9c8fb/runs/heroic_quill_wmzm404n5n?wsid=/subscriptions/96aede12-2f73-41cb-b983-6d11a904839b/resourceGroups/sdk/providers/Microsoft.MachineLearningServices/workspaces/sdk-canary